packages/design-system-docs/src/pages/components/Table/table.example.html

```diff
@@ -2,7 +2,7 @@
 <% [ 
   { title: 'Default table', className: 'ds-c-table'}, 
   { title: 'Borderless table', className: 'ds-c-table--borderless'}, 
-  { title: 'Striped table', className: 'ds-c-table--striped'}, 
+  { title: 'Striped and densed table', className: 'ds-c-table--striped ds-c-table--dense'}, 
```

Comment:
Should we remove the modifier code here and replace with full HTML for all examples since we are trying to get away from that?

Reply:
This isnt exactly modifier code, this is just the templating language we use, its called EJS.